### ISSUE TITLE
Customizing CSS - It doesn't work

### ISSUE THREAD

**Issue by Nexsuperne101** [blockscout/blockscout#8662]
### Description

Hi. I am trying to change the colors in _poa_ to black and red. I have followed the instructions, stopped blockscout, updated the _variables.scss and also _poavariables.scss. I then run the webpack which takes a couple of minutes and then restart the blockscout service. I then refresh the webpage to see it is exactly as it was before. I have managed some minor logo alterations, but that's it so far. Can you please advise. Is it a bug, or am I missing something?

Thanks, Nex

### Type of the installation

Manual from the source code

### Type of the JSON RPC archive node

polygon-edge RPC

### Type of the chain

L1

### Backend version

5.2.3

### Elixir & Erlang/OTP versions

1.13

### Operating system

Linux Ubuntu 20.04 server

**Comment by vbaranov** [comment]
@Nexsuperne101 you should run `mix phx.digest` at the end in order apply changes.

**Comment by Nexsuperne101** [comment]
perfect!!! Thanks. I couldn't find this in the instructions. I'm not a coder, so is it something that you guys just know?

**Comment by vbaranov** [comment]
@Nexsuperne101 please let us know which instruction did you follow? And we'll add a missing part, if it is.

**Comment by Nexsuperne101** [comment]
I think I missed a bit. My process was as follows;

Stop the block explorer service, go to /usr/local/blockscout/apps/block_scout_web/assets
In filezilla I then modified the _variables,scss and _poa_variables.scss with the new colour scheme. I then ran sudo npm run-script deploy and then went back to /usr/local/blockscout. I then ran mix phx.digest and finally restarted the block explorer service. This method now works. My errors were down to being a noob..... I am now changing the colour scheme and a bit of the css to suit our branding, as our website is black, red and grey.

Thanks for your help, it is much appreciated!
